app/services/order_details/dispute_resolved_notifier.rb

```diff
@@ -4,12 +4,20 @@ module OrderDetails
 
   class DisputeResolvedNotifier < SimpleDelegator
 
+    attr_accessor :current_user
+
+    def initialize(order_detail, current_user='')
```

Comment:
I think this is a good place for a keyword arg. And also, using a a blank string as the default seems a little weird here. Maybe just a nil.

```suggestion
    def initialize(order_detail, current_user: nil)
```